_episodes/markdown.md

```diff
@@ -88,7 +88,7 @@ Repo for learning how to make websites with Jekyll pages
 Vanilla text may contain *italic* and **bold words**.
 
 This paragraph is separated from the previous one by a blank line.
-Line breaks  
+Line breaks
```

Comment:
This one has been going back and forth. Do we not want the two trailing spaces here?